### DISCUSSION TITLE
What is the difference between workflows v1 and subflows

### DISCUSSION THREAD

**Discussion by KylinMountain** [microsoft/graphrag#1264] [General]
It confuse me....
![image](https://github.com/user-attachments/assets/42ac0647-9271-4ba9-adce-8281bf331236)

**Comment by AlonsoGuevara** [comment]
Hi! It is part of an effort we are doing to simplify and reduce GraphRAG's architecture.
You'll see a lot of these become deprecated eventually as we make progress.

**Reply by KylinMountain** [reply]
Great, hope the new architecture will be easy to customized.
